src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
```

Comment:
Why the generated folder name is `emailcommunication`? It shoudl be `communication\email` if directly generated by aaz-dev-tools. Please do not manually change the folder of `aaz`, it should be directly generated from `aaz-dev-tools`.

Reply:
Thank you. This has been updated

Reply:
Hello @kairu-ms. Please let me know if you have any further questions or concerns. Thank you

Reply:
Hello @AllyW 
Please let me know if you need additional information. Thank you